spec/lib/linters/spectral_spec.rb

```diff
@@ -5,7 +5,7 @@
   subject { described_class.new(file:) }
 
   it 'executes the spectral lint command' do
-    expect(subject).to receive(:`).with("spectral lint -f json \"/tmp/file.txt\"").and_return("{}")
+    expect(subject).to receive(:`).with("npx spectral lint -f json \"/tmp/file.txt\"").and_return("{}")
```

Comment:
If npx has to be used then we need to document this in the README and make sure it is on our heroku server